docs/operating-scylla/nodetool-commands/tasks/status.rst

```diff
@@ -0,0 +1,47 @@
+Nodetool tasks status
+=========================
+**tasks status** - Gets the status of a task manager task. If the task was finished it is unregistered.
+
+Syntax
+-------
+.. code-block:: console
+
+   nodetool tasks status <task_id>
+
+For example:
+
+.. code-block:: shell
+
+   > nodetool tasks status ef1b7a61-66c8-494c-bb03-6f65724e6eee
+
+Example output
+--------------
+
+.. code-block:: shell
+
+   id : 52b69817-aac7-47bf-9d8d-0b487dd6866a
```

Comment:
nit: why the column separated from the caption by space?
typically, I'm used to this looking like `id: 52b69817-aac7-47bf-9d8d-0b487dd6866a`.